lib/cli/render-version.js

```diff
@@ -22,7 +22,5 @@ module.exports = async () => {
     return '';
   })();
 
-  writeText(
-    `Framework Core: ${version}${installationModePostfix}${globalInstallationPostfix}`,
-  );
```

Comment:
changed so that it's easy to confirm the fork is used instead of the original CLI.